include/title_screen.h

```diff
@@ -0,0 +1,114 @@
+#ifndef GUARD_TITLE_H
+#define GUARD_TITLE_H
+
+#include "global.h"
+
+struct TitleStruct;
+
+typedef void (*TitleFunc)(struct TitleStruct*);
+
+struct TitleStruct {
+    u32 unk0;
+    u32 unk4;
+    u32 unk8;
+    u8 fillerC[4];
+    struct Unk10{
```

Comment:
1) space between `Unk10` & `{`
2) use a better name, like `TitleStructUnk10`, because it's still visible in global scope